src/connectors/argentMobile/modal/starknet/adapter.ts

```diff
@@ -50,9 +50,9 @@ export class StarknetAdapter
   // NamespaceAdapter
   public namespace = "starknet"
   public methods = [
```

Comment:
we need to be backwards compatible, so to the outside world we should expose `wallet_*` methods, but to the mobile client via WC `starknet_*` requests should be send

Reply:
@kamilargent just to be clear
atm we'll accept `wallet_` and, in starknetkit, rename that to `starknet_` for this current release?

Reply:
Yes, to the Dapp developer it should look like we use `wallet_requestAddInvokeTransaction` but internally to WC everything with `starknet_` will be send. And it will have to be kept like that for a few months.

Reply:
@kamilargent updated